docs/components/arm/eva.md

```diff
@@ -12,7 +12,9 @@ aliases:
 # SMEs: Bucket, Motion
 ---
 
-Configure an `eva` arm to integrate a [Automata Eva](https://automata.tech/products/hardware/about-eva/) robotic arm into your machine:
+Configure an `eva` arm to integrate an [Automata Eva](https://automata.tech/products/hardware/about-eva/) robotic arm into your machine.
+Make sure to first connect the arm to your machine's computer and turn it on.
```

Comment:
Maybe something like "Connect your arm to your machine and turn it on if you want to test your arm as  you configure it" would address @JessamyT 's feedback? @JessamyT what do you think. (Sierra wait to apply until we've discussed it)

Reply:
I like that because it tells the user why they would want to have it on, and avoids the confusion of them thinking that it's not possible to configure without it being powered on.

Reply:
applied